edg/parts/Lcd_Er_Tft1_28_3.py

```diff
@@ -0,0 +1,105 @@
+from edg.parts.PassiveConnector_Fpc import Fpc050BottomFlip
+from edg import *
+from edg.abstract_parts import *
+
+
+class Er_Tft_128_3_Outline(InternalSubcircuit, FootprintBlock):
+    """Footprint for TFT panel outline"""
+
+    def contents(self) -> None:
+        super().contents()
+        self.footprint('U', 'edg:Lcd_Er_Tft1_28_3_Outline', {},
```

Comment:
like htis is okay?

Reply:
Should be fine.

I typically put the FPC on User.Comments (see the ones I've made) since the user has flexibility into how the place the FPC. If the datasheet shows the FPC tucked under, I use that the position the FPC in the User.Comments layer. I use a dot for pin 1.

Reply:
![image](https://github.com/user-attachments/assets/df085529-b4e6-44dc-a31e-0046d8cbc659)

Reply:
Like this?

Reply:
Yep, looks good!